test-suite-groovy/src/test/groovy/io/micronaut/docs/server/urisyntax/MalformedUriSpec.groovy

```diff
@@ -14,14 +14,14 @@ class MalformedUriSpec extends Specification {
     @AutoCleanup
     EmbeddedServer embeddedServer = ApplicationContext.run(EmbeddedServer);
 
-    void testMalformedUriReturns400() {
+    void testMalformedUriReturns400AndUserInputNotPrintedInLogs() {
         when:
-        HttpURLConnection connection = (HttpURLConnection) new URL("$embeddedServer.URL/malformed/[]").openConnection()
+        HttpURLConnection connection = (HttpURLConnection) new URL("$embeddedServer.URL?\"><tag>=/malformed[]").openConnection()
         connection.connect()
 
         then:
         connection.getResponseCode() == 400
         connection.getResponseMessage() == HttpStatus.BAD_REQUEST.reason
-        connection.getErrorStream().getText().contains('"message":"Malformed URI:')
+        connection.getErrorStream().getText().contains('"message":"Malformed URI')
```

Comment:
I think we should add a closing quote `"` to the string we are checking for as that's the complete message now

```suggestion
        connection.getErrorStream().getText().contains('"message":"Malformed URI"')
```

Reply:
As it's a tiny change I committed it 😄 

Thanks for this @be-a-bee 👍